docs/Configuration.md

```diff
@@ -291,13 +320,15 @@ batch pool of tests, but is used in combination of
 
 ### lock_on_queue
 
-An integer.
+A non-negative integer. Max length 9 digits.
+
 A label to associate a test to a specific Test Agent.
 Default value: `0`.
 
 ### age_reuse_previous_test
 
-A positive integer.
+A positive integer. Max length 9 digits.
+
 The shelf life of a test in seconds after its creation.
 Default value: `600`.
```

Comment:
That is too high. 5 digits will give room for extreme values. We get less to test.